packages/crayons-core/src/components/alert/alert.tsx

```diff
@@ -0,0 +1,157 @@
+import {
+  Component,
+  Prop,
+  h,
+  Host,
+  Watch,
+  Element,
+  Method,
+  EventEmitter,
+  Event,
+} from '@stencil/core';
+
+const iconMap = {
+  error: 'alert',
+  warning: 'info',
+  info: 'info',
+  success: 'success',
+};
+
+const iconColorMap = {
+  error: '#E43538',
+  warning: '#C7502F ',
+  info: '#264966',
+  success: '#00795B',
+};
```

Comment:
These colours can you take from the DSM which has been specified for buttons? Then all colours will be standard DSM colours

Reply:
In DSM, Button variations doesn't have these colours kaushal. i..e there is no success or warning or info type buttons